spec/models/adventure_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe Adventure do
+  subject { FactoryGirl.build(:adventure) }
+  it { should belong_to(:bucket_list) }
+
+  it { should have_valid(:name).when('Visit Sue', 'Eat alligator') } ||
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.